bridge/test/dutchAuction/test.ts

```diff
@@ -0,0 +1,97 @@
+import { expect } from "chai";
+import { ethers, network } from "hardhat";
+import { DutchAuction } from "../../typechain-types";
+
+import {
+  deployStaticWithFactory,
+  factoryCallAny,
+  factoryCallAnyFixture,
+  Fixture,
+  getFixture,
+} from "../setup";
+
+let fixture: Fixture;
+let dutchAuction: DutchAuction;
+const validators: String[] = [];
+const stakingTokenIds: String[] = [];
+
+describe("Testing Dutch Auction", async () => {
+  beforeEach(async function () {
+    fixture = await getFixture(true, false, false);
+    validators.push("0x0000000000000000000000000000000000000000");
+    validators.push("0x0000000000000000000000000000000000000001");
+    validators.push("0x0000000000000000000000000000000000000002");
+    validators.push("0x0000000000000000000000000000000000000003");
+    await factoryCallAnyFixture(
+      fixture,
+      "validatorPool",
+      "registerValidators",
+      [validators, stakingTokenIds]
+    );
+    dutchAuction = (await deployStaticWithFactory(
+      fixture.factory,
+      "DutchAuction",
+      "DutchAuction",
+      undefined
+    )) as DutchAuction;
+  });
+
+  it("Should obtain bid price at first auction block", async () => {
+    expect(Number(await dutchAuction.getPrice())).to.be.equal(
+      Number("10000950400000000000000")
+    );
+  });
+
+  it("Should obtain prices through five blocks according to dutch auction curve", async () => {
+    expect(Number(await dutchAuction.getPrice())).to.be.equal(
+      Number("10000950400000000000000")
```

Comment:
Magic numbers - I'd suggest extracting them into `const` variables with meaningful names, this one in particular as i can see it's reused a few places in the test, perhaps something like `const EXPECTED_PRICE_BLOCK_ZERO : Number = Number("10000950400000000000000")` and you can extract the others following the naming suit for the consequent blocks

Reply:
Nice. Implemented as suggested. Thanks!